docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+
+x-common-variables: &common-variables
```

Comment:
let's move this to a `.env` file. It what's most people do for envvars

Reply:
i reverted to the previous longhand